theme/lumo/vaadin-grid-tree-toggle-styles.html

```diff
@@ -67,6 +67,31 @@
         background-size: var(--vaadin-grid-tree-toggle-level-offset) var(--vaadin-grid-tree-toggle-level-offset);
         background-position: calc(var(--vaadin-grid-tree-toggle-level-offset) / 2 - 2px) 0;
       }
+
+      /* RTL specific styles */
+
+      :host([dir="rtl"]) {
+        margin-left: 0;
+        margin-right: calc(var(--lumo-space-s) * -1);
+      }
+
+      :host([dir="rtl"]) [part="toggle"] {
+        margin-left: 0;
```

Comment:
`margin-right` is not reset here. It could be reset or `margin-right` could be changed inside a `:host(:not([dir="rtl"])) [part="toggle"]` selector.

Reply:
The default value of `margin` is 0. I am just ensuring it with this line for the left side. 
No need in reseting it for the `right` side.

Reply:
Isn't `margin` being set here:
https://github.com/vaadin/vaadin-grid/pull/1706/files#diff-7c687aff3e311564441142d7b52e6161R28-R29

Reply:
Good catch! Updated with `:not([dir="rtl"])`